src/Simple.OData.Client.UnitTests/Core/ExpansionTests.cs

```diff
@@ -467,8 +467,8 @@ public async Task ExpandFunctionMultipleLevelsWithSelect()
                 "$expand=Product($expand=ProductCategory($expand=Category($expand=CategorySalesArea;$select=Code);$select=IsPrimary);$select=Id,ManufacturerId)," +
                 "ClientProductSkuPriceList($select=CurrencyId)," +
                 "ClientProductSkuSalesArea," +
-                "Product($expand=SupplierProductSkuClient($expand=SupplierProductSku($expand=SupplierProductSkuPriceList($expand=SupplierPriceList)));$select=Id,ManufacturerId)," +
-                "Product($expand=SupplierProductSkuClient($expand=SupplierProductSku($expand=SupplierProductSkuOnHand($expand=Warehouse)));$select=Id,ManufacturerId)&" +
+                "Product($expand=SupplierProductSkuClient($expand=SupplierProductSku($expand=SupplierProductSkuPriceList($expand=SupplierPriceList)));$expand=ProductCategory($expand=Category($select=Code);$select=IsPrimary);$select=Id,ManufacturerId)," +
+                "Product($expand=SupplierProductSkuClient($expand=SupplierProductSku($expand=SupplierProductSkuOnHand($expand=Warehouse)));$expand=ProductCategory($expand=Category($select=Code);$select=IsPrimary);$select=Id,ManufacturerId)&" +
```

Comment:
Why was this necessary? Is this a breaking change?

Reply:
sorry, I was wrong. I will revert my changes and fix this test.

Reply:
I have fixed this test, but I had to change the expected command text. According to the OData specification "A property MUST NOT appear in more than one expand item" all expand associations from each level will be merged.